src/Commons/QuaViewMonad.hs

```diff
@@ -109,6 +109,7 @@ initQuaViewContext quaViewEvents = do
       $ liftIO . initLogInfo "Updated settings"
 
     quaViewUserMessageHandlers <- liftIO . newIORef $ UserMessageCallback defaultMsgFun
+    quaViewUserPopupHandlers   <- liftIO . newIORef $ UserSingleMsgCallback defaultSingleMsgFun
```

Comment:
Can't you re-use defaultMsgFun here?

Reply:
I think not because of the same reason: `() ~ t` does not hold here.

Reply:
Ah yes... actually we can.